README.md

```diff
@@ -1,7 +1,26 @@
-sminnee/callbacklist
+# sminnee/callbacklist
+[![Build Status](https://travis-ci.org/sminnee/callbacklist.svg?branch=master)](https://travis-ci.org/sminnee/callbacklist)
+[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/sminnee/callbacklist/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/sminnee/callbacklist/?branch=master)
+[![codecov.io](https://codecov.io/github/sminnee/callbacklist/coverage.svg?branch=master)](https://codecov.io/github/sminnee/callbacklist?branch=master)
+
+
+[![Latest Stable Version](https://poser.pugx.org/sminnee/callbacklist/version)](https://packagist.org/packages/sminnee/callbacklist)
+[![Latest Unstable Version](https://poser.pugx.org/sminnee/callbacklist/v/unstable)](//packagist.org/packages/sminnee/callbacklist)
+[![Total Downloads](https://poser.pugx.org/sminnee/callbacklist/downloads)](https://packagist.org/packages/sminnee/callbacklist)
+[![License](https://poser.pugx.org/sminnee/callbacklist/license)](https://packagist.org/packages/sminnee/callbacklist)
+[![Monthly Downloads](https://poser.pugx.org/sminnee/callbacklist/d/monthly)](https://packagist.org/packages/sminnee/callbacklist)
+[![Daily Downloads](https://poser.pugx.org/sminnee/callbacklist/d/daily)](https://packagist.org/packages/sminnee/callbacklist)
+[![composer.lock](https://poser.pugx.org/sminnee/callbacklist/composerlock)](https://packagist.org/packages/sminnee/callbacklist)
```

Comment:
The number of badges is a little overwhelming to me. Maybe we remove:

 * Total Downloads, Daily Downloads (monthly is the best indication of module use)
 * composer.lock - the package neither has nor needs one
 * Latest unstable version - it's broken and stable is more valuable
 * Repo size - code size is more important